scripts/swarm-fix-route_manager.sh

```diff
@@ -399,18 +416,18 @@ function main {
       a)
       echo "STEP 8:"
       echo "Updating <<$INGRESS_NS>> namespace..."
-      echo "---> adding NAT UDP 514 exception... "
-      add_514_nat_exception $INGRESS_NS $OVERLAY_NET_1_PREFIX
+      echo "---> adding NAT UDP/514 UDP/162 TCP/6514 exception... "
+      add_nat_exception $INGRESS_NS $OVERLAY_NET_1_PREFIX
       echo ""
       ;;
 
       s)
       echo "STEP 8:"
       echo "NAT EXCEPTIONS LIST:"
-      echo "---> show NAT UDP 514 exceptions in <<$INGRESS_NS>> namespace:"
+      echo "---> show NAT UDP/514 UDP/162 TCP/6514 exceptions in <<$INGRESS_NS>> namespace:"
```

Comment:
echo "---> show NAT UDP/514 UDP/162 TCP/514 TCP/6514 exceptions in <<$INGRESS_NS>> namespace:"